vm/systemSmartContracts/eei.go

```diff
@@ -284,11 +315,6 @@ func (host *vmContext) DeploySystemSC(
 	host.SetSCAddress(oldSCAddress)
 	host.addContractDeployToOutput(newAddress, ownerAddress, baseContract)
 
-	err = host.systemContracts.Replace(newAddress, contract)
```

Comment:
This operation is not needed anymore?

Reply:
this operation was problematic as the container thus not have rollbck and start in epoch or bootstrap capabilities.

Reply:
ok